workflow/slurm/config.yaml

```diff
@@ -1,19 +1,22 @@
 cluster:
   "sbatch
     --partition={resources.partition}
-    --cpus-per-task={threads}
+    --cpus-per-task={resources.threads}
     --mem={resources.mem_mb}
     --job-name=smk-{rule}
     --gres=gpu:{resources.gpu}
-    --output=logs/%j" # you can update where the logs will be saved (absolute path, or relative to /.../sopa/workflow)
+    --time={resources.time}
+    --output=logs/{rule}_%j.log" # you can update where the logs will be saved (absolute path, or relative to /.../sopa/workflow)
 default-resources:
-  - partition=shortq
-  - mem_mb=32000
+  - partition=shortq # use the partition/queue name on your cluster
+  - mem_mb=15000
   - gpu=0
+  - threads=8
+  - time="30:00:00"
```

Comment:
Minor comment: this might be something that doesn't fit well for all users. I think this is something that the user should add if needed. WDYT?